lib/resources/example_json/dstu2_examples_relatedperson.rb

```diff
@@ -183,7 +183,7 @@ module Resources
               }
             ],
             "patient": {
-              "reference": "Patient/3838010",
+              "reference": "Patient/3838011",
               "display": "Peters, Jennifer"
```

Comment:
So... I know that this is actually what is returned by the url in question here. But it is probably a very confusing example to give, since the patient's name in the description is TIMOTHY and here is Jennifer. Probably just want to change this to Timothy even though that's different than what people will see if they fire off the request themselves. Or find another patient to use as an example